docs/variants/novacustom_ns5x_7x/building-manual.md

```diff
@@ -31,7 +31,7 @@ The easiest way to build coreboot is to use the official Docker image.
 1. Checkout to the desired Dasharo revision:
 
     > Replace the REVISION with one of the:
-    > - `novacustom_ns5x/release` for the latest released version
+    > - `clevo/release` for the latest released version
```

Comment:
@mkopec Have you actually tried to follow thos documentation? I think that the real problem is that there is no `build.sh` script on the `clevo/release` branch.
Maybe we should fall back to using a few commands, as we have for Optiplex, for example: https://github.com/Dasharo/docs/blob/master/docs/variants/dell_optiplex/building-manual.md
Unless there are better suggestions.

Reply:
@mkopec ?

Reply:
Fixed